bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/config/RFZoneConfig.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.alarmdecoder.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link RFZoneConfig} class contains fields mapping thing configuration parameters for RFZoneHandler.
+ *
+ * @author Bob Adair - Initial contribution
+ */
+@NonNullByDefault
+public class RFZoneConfig {
+    public int serial = -1;
```

Comment:
After reading the code below, this seems to be an invalid value. Leave it out if that is correct

Reply:
Yes, I think that is a recent change. I initialize it to an invalid value (0 is potentially a valid value) rather than using a @Nullable Integer to determine if it was not set by the user. As a general rule, anything weird like this in the code was done to get rid of  null annotation warnings. I think here it isn't strictly necessary because that parameter should be marked as required, but it still seemed like a good idea to have the code enforce it.